shodan/client.py

```diff
@@ -502,9 +502,12 @@ def search(self, query, page=1, limit=None, offset=None, facets=None, minify=Tru
         if facets:
             args['facets'] = create_facet_string(facets)
 
+        if timeout:
+            args['timeout'] = timeout
```

Comment:
``timeout`` isn't a parameter that the Shodan API uses. Does this actually change the timeout?

Reply:
I removed timeout, its supposed to be used directly in requset.session object, but i made a mistake, and in my tests there is no need to use timeout, with incremental delay i solved the problems.